src/types.ts

```diff
@@ -2,150 +2,86 @@
  *          IMPORTS          *
  *****************************/
 
+import { Greeks, Option, Position } from "@arrow-markets/arrow-common-sdk"
+import {
+  ContractType,
+  OrderType
+} from "@arrow-markets/arrow-common-sdk/lib/types/option"
 import { ethers } from "ethers"
 
 /**********************************
  *          USEFUL TYPES          *
  **********************************/
 
 export enum Version {
-    V4 = "v4",
```

Comment:
Just have to say how sad it is to go to 2 space tabs because it's so squished haha

Reply:
Definitely not the end of the world though